src/ui/components/SecondaryToolbox/ReduxDevTools.tsx

```diff
@@ -75,35 +75,50 @@ export const ReduxDevToolsPanel = () => {
     const rootComponent = rootRef.current;
     const store = rootComponent.store!;
 
-    const matchingAnnotationsByTimeRange = reduxAnnotations.filter(annotation => {
-      return currentTimestamp != null && annotation.time < currentTimestamp;
+    const allInstanceIds: string[] = ["default"];
```

Comment:
Could you leave a comment about what the significance of the "default" instance ID is?

It's not obvious to me what "default" or "INIT_INSTANCE" means really.

Reply:
These are all internals of the Redux DevTools "instances" reducer.

The Redux DevTools UI has its own Redux store with its own reducers.  The primary part that we care about is the slice reducer that manages which "store instances" are connected, and calculates the "lifted state" for each store (ie, "after 3 increment actions, the state is `{value: 3}`".  See [`reducers/instances.ts` in the Redux DevTools repo for the logic](https://github.com/reduxjs/redux-devtools/blob/b82de745928211cd9b7daa7a61b197ad9e11ec36/packages/redux-devtools-app/src/reducers/instances.ts).

The RDT internals first pass along an `INIT_INSTANCE` action to let the UI know that a given store instance exists, and then later pass along the serialized actions for processing to update that instance's calculated state.

Internally, the `instances` reducer appears to have handling for a "default" state, and falls back to that value if no instances are connected.  In my experimenting I found I needed to dispatch an action related to that as part of resetting the internal state.